src/tyssue/geometry/planar_geometry.py

```diff
@@ -42,6 +44,25 @@ def update_areas(sheet):
         sheet.edge_df["sub_area"] = sheet.edge_df["nz"] / 2
         sheet.face_df["area"] = sheet.sum_face(sheet.edge_df["sub_area"])
 
+    # @staticmethod
+    # def update_repulsion(sheet):
+    #     # Create globale grid
+    #     grid = np.mgrid[np.min(sheet.vert_df['x']) - 0.1:np.max(sheet.vert_df['x']) + 0.1:0.1,
+    #            np.min(sheet.vert_df['y']) - 0.1:np.max(sheet.vert_df['y']) + 0.1:0.1]
+    #     face_repulsion = gaussian_repulsion(grid, sheet)
+
+    #     sheet.vert_df['v_repulsion'] = 0
+    #     sheet.vert_df['grid'] = 0
```

Comment:
If you comment this method due to pandas error, it is possible to fix it with that : 
```
sheet.vert_df['v_repulsion'] = sheet.vert_df['v_repulsion'].astype(object)
sheet.vert_df['grid'] = sheet.vert_df['grid'].astype(object)
```

Reply:
Thanks a lot Sophie
can you push that fix on this branch (I think you have the right to do so)?

If you have a look at the tests, I commented out some of yours, could you put them back?

Thanks a lot!

G